charts/gha-runner-scale-set/templates/manager_role.yaml

```diff
@@ -22,6 +30,7 @@ rules:
   - ""
   resources:
   - secrets
+  - serviceaccounts
```

Comment:
may be duplicate the section only for serviceaccounts?
like we have 2 sections for role and rolebinding.